mesonbuild/dependencies/base.py

```diff
@@ -576,37 +576,67 @@ def _set_cargs(self):
         self.compile_args = self._convert_mingw_paths(shlex.split(out))
 
     def _search_libs(self, out, out_raw):
```

Comment:
can we rename `out` and `out_raw` to something that gives a clue what they contain?

Reply:
out means "pkg-config output" and out_raw means "pkg-config output without PKG_CONFIG_ALLOW_SYSTEM_LIBS". I documented it in the docstring what each means.

Reply:
There is, but as instance variable i think it's even worse, because it does have even less context. 'raw' just mean, without something or some processing that is unspecified...

Reply:
So I was making the change when I realized that it'll touch a lot of other code unnecessarily, since `out` and `out_raw` is used elsewhere too. Let's not churn the CI unnecessarily, since this PR has to go into the stable branch.